assemblies/assembly-rhdh-integration-aks.adoc

```diff
@@ -13,6 +12,8 @@ include::modules/admin/proc-rhdh-deploy-aks.adoc[leveloffset=+1]
 
 include::modules/admin/proc-rhdh-deploy-aks-using-helm.adoc[leveloffset=+2]
 
+include::../artifacts/snip-install-helm-cli-k8s.adoc[leveloffset=+3]
```

Comment:
```suggestion
include::modules/installation/proc-installing-rhdh-on-kubernetes-with-helm-cli.adoc[leveloffset=+3]
```

Reply:
Doing it this way caused a build error, which is why the snippet was created